uk.co.compucorp.civicrm.hremails/email-templates/src/partials/request-data.html

```diff
@@ -0,0 +1,10 @@
+<row class="{{class}}">
+  <columns class="request-data-key" large="{{#if keySize}}{{keySize}}{{else}}3{{/if}}">
```

Comment:
The if condition is very hard to read, do we have any other alternatives of writing the same?